corehq/apps/couch_sql_migration/diff.py

```diff
@@ -317,33 +304,6 @@ def _filter_xform_id_diffs(couch_case, sql_case, diffs):
     return remaining_diffs
 
 
-def _filter_case_attachment_diffs(couch_case, sql_case, diffs):
-    """Attachment JSON format is different between Couch and SQL"""
-    remaining_diffs = [diff for diff in diffs if diff.path[0] != 'case_attachments']
-    if len(remaining_diffs) != len(diffs):
-        couch_attachments = couch_case.get('case_attachments', {})
-        sql_attachments = sql_case.get('case_attachments', {})
-
-        for name, couch_att in couch_attachments.items():
-            sql_att = sql_attachments.get(name, Ellipsis)
-            if sql_att == Ellipsis:
-                remaining_diffs.append(FormJsonDiff(
-                    diff_type='missing', path=('case_attachments', name),
-                    old_value=couch_att, new_value=sql_att
-                ))
-            else:
-                att_diffs = json_diff(couch_att, sql_att)
-                filtered = _filter_partial_matches(att_diffs, PARTIAL_DIFFS['case_attachment'])
-                filtered = _filter_renamed_fields(filtered, couch_att, sql_att, 'case_attachment')
-                for diff in filtered:
-                    diff_dict = diff._asdict()
-                    # convert the path back to what it should be
-                    diff_dict['path'] = tuple(['case_attachments', name] + list(diff.path))
-                    remaining_diffs.append(FormJsonDiff(**diff_dict))
-
-    return remaining_diffs
-
-
```

Comment:
@jmtroth0 I think you may need to update the diff filtering to completely ignore case attachment diffs.